hash_practice/exercises.py

```diff
@@ -2,18 +2,53 @@
 def grouped_anagrams(strings):

```

Comment:
✨ However we can say the time complexity is actually O(n) assuming words are limited in length (as they would be in English language). n would represent the length of the entire strings array. The join and sort functions only operate on a single string of negligible length (assuming valid English words) so they don't necessarily need to be accounted for in the overall time complexity.